edu-services/gradebook-service/hibernate/src/java/org/sakaiproject/tool/gradebook/Gradebook.java

```diff
@@ -23,6 +23,9 @@
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 
+import lombok.Getter;
```

Comment:
Same maybe lombok @Data is a better fit here?

Reply:
Done